playbooks/provision.yml

```diff
@@ -115,6 +115,12 @@
       tags: logrotate
 
   tasks:
+    - name: Fix Ruby # noqa 301
+      command: bash -lc "rbenv uninstall -f {{ ruby_version }} && rbenv install {{ ruby_version }}"
+      become: yes
+      become_user: "{{ app_user }}"
+      when: ansible_distribution_major_version >= '20'
```

Comment:
Hm, this would happen each time we run the provisioning and it takes a while. Did you try to increase the memory of the virtual machine?

Or maybe we can skip the task if assets have compiled successfully? That would be a weird hack here, too...

Reply:
Thanks for the suggestion!

I did try to increase memory and cpu just in case. Unfortunately the error persisted. My best guess for the cause of this error still is something within the galaxy module taking care of installing ruby.

I think checking if assets precompilation work before running this would be hard and confusing, specially because the precompile runs in a different playbook (deploy.yml).

I understand that this should run every time a new ruby gets installed. What do you think about setting a hidden file with `{{ ruby_version}}` and only running the fix task if the task setting such file changed? This would prevent it to run the fix when not necessary.

Reply:
> What do you think about setting a hidden file

Yes, it's also a hack but we are hacking anyway. :wink: It's a good idea. If you can easily put it in the ruby installation path then it gets removed automatically when removing a ruby version. It's like marking an installation as "patched". But if that's too difficult then a simple file in the home directory would do, too.

Reply:
Done in commit 415c8ffcf9950458864145e2fbb2987f024883a4